integration-cli/docker_api_stats_test.go

```diff
@@ -69,3 +71,41 @@ func (s *DockerSuite) TestStoppedContainerStatsGoroutines(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestApiNetworkStats(c *check.C) {
+	// Run container for 30 secs
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "sleep 30")
+	id := strings.TrimSpace(out)
+	err := waitRun(id)
+	c.Assert(err, check.IsNil)
+
+	// Retrieve the container address
+	contIP := findContainerIP(c, id)
+	numPings := 10
+
+	// Get the container networking stats before and after pinging the container
+	nwStatsPre := getNetworkStats(c, id)
+	_, err = exec.Command("ping", contIP, "-c", strconv.Itoa(numPings)).Output()
```

Comment:
Thought of that, but preferred not to spawn another container when pinging from the host is all what is needed here.
Could you elaborate more on why it would be better to ping from another container.